content/post/agile-analytics-p2.md

```diff
@@ -0,0 +1,56 @@
+* Business-as-usual support
+
+I often say that analytics is a discipline that's half software engineering and half research. The parts of scrum that don't work so well for analytics are the parts that are more aligned with the "research" half of the analytics discipline.
+
+## The fortuitous finding
+
+Often, some of the highest value insights that an analytics team generate come from someone seeing something interesting -- but totally unrelated to their current assignment -- and then pursuing it.  This type of curiosity and tenacity is something we *love* to see in analysts we hire, but the scrum process, when it becomes a well-oiled box-checking process, can squash these types of endeavors.
+
+When you hear something like "Ooh shoot. I noticed that issue before but I was so focused on finishing my story before the sprint ended that I didn't look closer" then you may be suffering from this. This "illusion of effectiveness", where the team feels like they're getting all their stories done (collecting those sweet, sweet story points) is pernicious because it *feels* productive despite missing out on potential high-value opportunities. The analytics team is *supposed* to be generating novel insights, which means that there has to be some tolerance for unplanned investigations.
+
+As a researcher, I've often found that the most valuable insights I've generated have come from investigating something that looked odd or interesting and was only tangentially related to the task at hand. These sorts of exploratory analysis make up the "science" part of data science and we want to ensure that our analysts have the time and the inclination to pursue these opportunities and generate novel insights.
+
+## Exploratory data analysis
+
+Exploratory data analysis is an important part of the "basic research" function of an analytics or data science team. Unfortunately, it's really hard to write "user stories" or "acceptance criteria" for this sort of work because ... the whole point is that you aren't sure what you're going to find, and it requires a researcher's intuition to know when the story is "done" (because there's always more things you *could* look into, but often those marginal investigations aren't valuable).
+
+The nature of exploratory data analysis means that the objectives of the analysis may change *as you do the work*. You sometimes realize "oh x isn't actually very interesting -- the real issue is understanding y." If your scrum process is too inflexible, you can end up expending effort against tasks that aren't valuable simply because they're required to get the story done.  While (software) engineering is a discipline of creatively applying mathematical tools to a well-defined problem, data science is about creatively applying mathematical tools to *poorly defined* problems, and it's this fundumental difference in nature that makes partitioning the work into small well-scoped tasks sometimes impossible for data science.
```

Comment:
Not sure if it's helpful, but my team uses [spikes](https://www.leadingagile.com/2014/04/dont-estimate-spikes/) for this type of work. Although your point remains in that it's difficult to come up with success criteria for exploratory work.

Reply:
Yep! I talk a bit about using spikes in part 3 :)